package.json

```diff
@@ -9,13 +9,33 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.8.2",
+    "@mui/material": "^5.8.2",
+    "@popperjs/core": "^2.11.6",
     "axios": "^0.27.2",
+    "classnames": "^2.3.1",
+    "lodash": "^4.17.21",
     "next": "12.1.6",
     "react": "18.1.0",
-    "react-dom": "18.1.0"
+    "react-dom": "18.1.0",
+    "react-paginate": "^8.1.3",
+    "react-popper": "^2.3.0",
+    "react-router-dom": "^5.3.3",
+    "recharts": "^2.1.13",
```

Comment:
`"recharts": "2.1.12",`
yarn upgrade recharts@2.1.12

Downgrade version of recharts, because it's has issue with imports